src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/_help.py

```diff
@@ -0,0 +1,206 @@
+    examples:
+        - name: Create a Skype Channel for a bot with messaging and screen sharing enabled
+          text: |-
+            az bot skype create -n botName -g MyResourceGroup --enable-messaging
+            --enable-screen-sharing
+"""
+helps['bot kik create'] = """
+    type: command
+    short-summary: Create a Kik Channel on a bot.
+    examples:
+        - name: Create a Kik Channel for a bot.
+          text: |-
+            az bot kik create -n botName -g MyResourceGroup -u mykikname
+            -p password --key key --is-validated
+"""
+helps['bot directline create'] = """
+    type: command
+    short-summary: Create a DirectLine Channel on a bot with only v3 protocol enabled.
+    examples:
+        - name: Create a DirectLine Channel for a bot.
+          text: |-
+            az bot directline create -n botName -g MyResourceGroup --disablev1
+"""
+helps['bot telegram create'] = """
+    type: command
+    short-summary: Create a Telegram Channel on a bot.
+    examples:
+        - name: Create a Telegram Channel for a bot.
+          text: |-
+            az bot telegram create -n botName -g MyResourceGroup --access-token token
+            --is-validated
+"""
+helps['bot sms create'] = """
+    type: command
+    short-summary: Create an SMS Channel on a bot.
+    examples:
+        - name: Create an SMS Channel for a bot.
+          text: |-
+            az bot sms create -n botName -g MyResourceGroup --account-sid sid
+            --auth-token token --is-validated
+"""
+helps['bot slack create'] = """
+    type: command
+    short-summary: Create a Slack Channel on a bot.
+    examples:
+        - name: Create a Slack Channel for a bot.
+          text: |-
+            az bot slack create -n botName -g MyResourceGroup --client-id clientid
+            --client-secret secret --verification-token token
+"""
+helps['bot authsetting'] = """
+    type: group
+    short-summary: Manage OAuth Connection Settings on a bot.
+"""
+helps['bot authsetting create'] = """
+    type: command
+    short-summary: Create an OAuth Connection Setting on a bot.
+    examples:
+        - name: Create a new OAuth Connection Setting on a bot.
+          text: |-
+            az bot authsetting create -g MyResourceGroup -n botName -c myConnectionName
+            --client-id clientId --client-secret secret --scopes "scope1 scope2" --service google
+            --parameters id=myid
+"""
+helps['bot authsetting show'] = """
+    type: command
+    short-summary: Show details of an OAuth Connection Setting on a bot.
+"""
+helps['bot authsetting list'] = """
+    type: command
+    short-summary: Show all OAuth Connection Settings on a bot.
+"""
+helps['bot authsetting delete'] = """
+    type: command
+    short-summary: Delete an OAuth Connection Setting on a bot.
+"""
+helps['bot authsetting list-providers'] = """
+    type: command
+    short-summary: List Details of All service Providers available for creating OAuth Connection Settings.
```

Comment:
Lots of unnecessary caps. `List details for all OAuth service providers available` maybe?